sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/RenderClientBuilder.java

```diff
@@ -284,6 +311,54 @@ private HttpPipeline createHttpPipeline() {
         return httpPipeline;
     }
 
+    /*
```

Comment:
What are these and what are they doing at the end of this class? I doubt this is compiling